spec.bs

```diff
@@ -917,6 +966,28 @@ To <dfn>validate and convert auction ad config</dfn> given an {{AuctionAdConfig}
       1. Set |auctionConfig|'s [=auction config/seller signals=] to the result of
         [=serializing a JavaScript value to a JSON string=], given |result|.
     * To handle an error, set |auctionConfig|'s [=auction config/seller signals=] to failure.
+1. If |config|["{{AuctionAdConfig/auctionNonce}}"] [=map/exists=]:
+  1. Let |nonce| be an empty [=string=].
+  1. [=list/For each=] |i| in [=the range=] from 0 to the [=string/length=] of
+    |config|["{{AuctionAdConfig/auctionNonce}}"], exclusive:

```

Comment:
Out of curiosity, why do we need to create a copy of the auctionNonce here, and why do we restrict that string to only characters NULL and those in the range space through delete? How is this distinct from the validation we apply in =get uuid from string=?

Reply:
I see. Thanks, and done.